docs/templates/resources/search_deployment.html.markdown.tmpl

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Should we follow the convention of placing templates within `./templates` folder? reference: https://github.com/hashicorp/terraform-plugin-docs?tab=readme-ov-file#conventional-paths.
For this example it would mean placing the file in `./templates/resources/search_deployment.html.markdown.tmpl`

Reply:
I don't have a strong opinion here. Updating

Reply:
updated